### PR TITLE
2 new sample pages showing off new Skuid Capabilities

### PR DESCRIPTION
**Pull request by Rnhatch**
New pages for the Sample Pages Repo

### CODE REVIEW

File: Skuid_Techniques/Form_Validation/Validation.xml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+											<renderConditions logictype="and"/>
+											<styleVariantConditions/>
+										</skuid__form>
+									</components>
+								</skuid__tabPanel>
+								<skuid__tabPanel name="Formulas">
+									<components>
+										<skuid__wrapper uniqueid="sk-RQd-35647">
+											<components>
+												<skuid__text contents="Validation Rules based on data Formulas.&amp;nbsp;&lt;br /&gt;&#10;- Field will be valid if formula is true. ​​​​​​&lt;br /&gt;&#10;- Formulas don&amp;#39;t automatically have field context.&amp;nbsp; You have to explicly say &amp;quot;Amount 1=100&amp;quot;.&amp;nbsp; &amp;quot;=100&amp;quot; will not work.&amp;nbsp;&amp;nbsp;" uniqueid="sk-RQd-35648">
```
Comment (itmightbecody) [inline Skuid_Techniques/Form_Validation/Validation.xml:89]:
```suggestion
												<skuid__text contents="Validation rules based on formulas:&lt;br /&gt;&#10;- These fields will be valid if their validation formula returns true. The expected value format is listed below as help text.&lt;br /&gt;&#10;- Note that validation formulas don&amp;#39;t automatically have field context. You must explicly say the field name, like in the &lt;strong&gt;Amount 2&lt;/strong&gt; validation :&lt;br /&gt;&#10;&lt;br /&gt;&#10;&amp;nbsp; VALUE({{Amount1}})&amp;lt;VALUE({{Amount2}})" uniqueid="sk-RQd-35648">
```

Clarity suggestions

File: Skuid_Techniques/Form_Validation/Validation.xml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+											<renderConditions logictype="and"/>
+											<styleVariantConditions/>
+										</skuid__form>
+									</components>
+								</skuid__tabPanel>
+								<skuid__tabPanel name="JS Snippet">
+									<components>
+										<skuid__wrapper uniqueid="sk-h5U-22430">
+											<components>
+												<skuid__text contents="Validation Rules based on Javascript Snippet&lt;br /&gt;&#10;- Field will be valid if Snippet returns true. ​​​​​​&lt;br /&gt;&#10;- Example only allows letters to be entered.&amp;nbsp;" uniqueid="sk-h5U-22431">
```
Comment (itmightbecody) [inline Skuid_Techniques/Form_Validation/Validation.xml:143]:
```suggestion
												<skuid__text contents="Validation rules based on a JavaScript snippet:&lt;br /&gt;&#10;- This field will be valid if the snippet returns true. ​​​​​​&lt;br /&gt;&#10;- This example only accepts letters as valid characters." uniqueid="sk-h5U-22431">
```

Formatting consistency + clarity

File: Skuid_Techniques/Form_Validation/Validation.xml [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+											</columns>
+											<styles>
+												<spacing/>
+											</styles>
+											<renderConditions logictype="and"/>
+											<styleVariantConditions/>
+										</skuid__form>
+									</components>
+								</skuid__tabPanel>
+								<skuid__tabPanel name="Password Form">
```
Comment (itmightbecody) [inline Skuid_Techniques/Form_Validation/Validation.xml:181]:
```suggestion
								<skuid__tabPanel name="Password Validation Example">
```

See comment below

**Review by itmightbecody — CHANGES_REQUESTED** [review]
The functionality here is great. Most of these suggestions are for clarity around the explanatory text in the pages. Let me know if you have any questions around these comments!

**Comment by Rnhatch** [comment]
Thanks @itmightbecody.  I've accepted all your suggestions.  Appreciate you partnership making this better.

File: Skuid_Techniques/Form_Validation/Validation.xml [hunk 4]
```diff
@@ -0,0 +1,281 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+	<models>
+		<model id="FormData" limit="20" query="true" datasource="Ui-Only" createrowifnonefound="true">
+			<fields>
+				<field id="URL" displaytype="URL" length="255" ogdisplaytype="TEXT"/>
+				<field id="Email" displaytype="EMAIL" length="255" ogdisplaytype="TEXT"/>
+				<field id="Social" displaytype="TEXT" length="255"/>
```
[the remark this answers is not in the document]
Comment (itmightbecody) [reply Skuid_Techniques/Form_Validation/Validation.xml:7]: @Rnhatch Is there a reason to not accept this change?

**Comment by Rnhatch** [comment]
@itmightbecody  three suggestions were hidden from me by Github.  Sorry for not catching them the first time.   I've committed them to the PR..  Thanks.

**Review by itmightbecody — APPROVED** [review]
Sneaky Github hiding suggestions. Looks good to me! Thank you for your work on this. 🎉

**Pull request merged**